annotation/src/main/resources/application.properties.EXAMPLE

```diff
@@ -15,6 +15,9 @@ mutationAssessor.url=http://mutationassessor.org/r3/?cm=var&var=VARIANT&frm=json
 # Comma separated list of isoform_overrides_resource:overrides_filename pairs.
 vep.isoform.overrides=uniprot:isoform_overrides_uniprot.txt,mskcc:isoform_overrides_at_mskcc.txt
 
+# Ensemble biomart PFAM filename
+ensemble.biomart.pfam=ensembl_biomart_pfam.txt
```

Comment:
@onursumer should add this fail to the PR

Reply:
@inodb We don't have `isoform_overrides_uniprot.txt`, `isoform_overrides_at_mskcc.txt` in the repository either. But I guess it makes sense to maintain the biomart file in this repository.

Reply:
@onursumer good point. I think we should actually store those and have some script to update/download them. Check if they're in sync. I'll create an issue